src/controllers/controllermanager.cpp

```diff
@@ -415,6 +415,12 @@ void ControllerManager::slotSavePresets(bool onlyActive) {
             continue;
         }
 
+        if (pPreset->filePath().startsWith(resourcePresetsPath(m_pConfig))) {
+            pPreset->setName(QString(tr("%1 (edited)")).arg(pPreset->name()));
```

Comment:
```suggestion
            pPreset->setName(pPreset->name + " (" + tr("edited") + ")");
```
Let's not bother translators with the formatting.